src/plugins/vis_type_timeseries/public/application/components/visualization.js

```diff
@@ -43,6 +43,10 @@ export function Visualization(props) {
   const { visData, model } = props;
   // Show the error panel
   const error = _.get(visData, `${model.id}.error`);
+  if (_.get(error, 'error.type') === 'index_not_found_exception') {
+    const index = _.get(error, 'error.index');
+    error.message = `Index "${index}" is missing`;
```

Comment:
Would be great to use translations here:
```suggestion
    error.message = i18n.translate('visTypeTimeseries.error.missingIndexErrorMessage', {
      defaultMessage: 'Index {index} is missing',
      values: { index }
    });
```

Reply:
Sure, updated the PR and its description